src/RunTimeException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class RunTimeException extends Exception
```

Comment:
RuntimeException exists in root namespace. avoid this name (same as above)

Reply:
This feedback belongs with PR #44.